app/views/bookmarks/_tools.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="share-widget" role="group" aria-label="Actions including with nested dropdown">
```

Comment:
so this is taking part of `_record_toolbar.html.erb` right?

Reply:
correct, and I added  `data-display="static"` to line 3 and `dropdown-menu-right dropdown-menu-md-left` classes to line 4.